pkg/agent/params.go

```diff
@@ -148,6 +148,9 @@ func assignKubernetesParametersFromAgentProfile(profile *datamodel.AgentPoolProf
 		addValue(parametersMap, "containerRuntime", profile.KubernetesConfig.ContainerRuntime)
 		if profile.KubernetesConfig.ContainerRuntime == "containerd" {
 			addValue(parametersMap, "cliTool", "ctr")
+			if url, k := profile.KubernetesConfig.ContainerRuntimeConfig["teleportdPluginURL"]; k {
+				addValue(parametersMap, "teleportdPluginURL", url)
+			}
```

Comment:
the same here, shall we move the url config to NodeConfiguration? Exposure to agentpool or container service concept is a legacy thing to AgentBaker, and should be limited.

Reply:
changed to use NodeBootstrapConfig. Idk why it's not being reflected here